stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/message/input/MessageInputViewStyle.kt

```diff
@@ -84,6 +91,12 @@ public data class MessageInputViewStyle(
     public val customCursorDrawable: Drawable?,
     public val attachmentMaxFileSize: Int,
     public val dividerBackground: Drawable,
+    public val pictureAttachmentIcon: Drawable,
+    public val pictureAttachmentIconTint: ColorStateList?,
```

Comment:
Is there any value by default in our sample? If we have something, then it's better to make these tint colors not-nullable